src/plugins/intel_gpu/src/graph/impls/ocl/arg_max_min.cpp

```diff
@@ -84,7 +84,10 @@ struct arg_max_min_impl : typed_primitive_impl_ocl<arg_max_min> {
             argm_params.argMaxMinSortType = kernel_selector::argm_sort::INDEX;
 
         if (outputs_num == 2) {
-            argm_params.inputs.push_back(convert_data_tensor(impl_param.input_layouts[2]));
+            if (primitive->input.size() == 3)
+                argm_params.inputs.push_back(convert_data_tensor(impl_param.input_layouts[2]));
```

Comment:
Can we classify this kind of branches with more representitive, so that we can clean up more easily when every thing is implemented in the future? 

For example, add some common handle like 
if(use_multiple_outputs()) {
...
} else {
...
}

I think, not only for this portion, but also for many other portions, we need to make effort to make it more distinguishable in the future, which code portions are to be refactored.

Reply:
agree with your suggestion. It has been updated accordingly.